docs/canarynet/canarywalletguide.md

```diff
@@ -0,0 +1,69 @@
+   
+   ![img](https://lh3.googleusercontent.com/TTrdCWUBYRO2zQ3HMNbowDO5xBdpV0AYsDBqn-bsUtJo5iTYRTJ1MAH9bPd-K57eJshDQOI2E5zn2P5YCu-Uf2124ntb57K0EuwsiweBvkOSE6Y9T3Sz7U9NAK7xlIq5rdzU5h8o=s0)
+
+4. You must then click on the plugin to create an account. This process is very easy and just requires you to record your seed phrase, which should be noted to ensure you have a way to restore control of your account, create a username and then a password. You will then have an account.*The account should look like this in the plugin.*
+
+   ![img](https://lh3.googleusercontent.com/FH_l2uUrM8Js6g9vH_06-Mg1hD7QruPT7FM4ve6CFZlMAVjtbjBwsluAVK0FnsWppwQSGWn_XtoPhQCSgInPTr5Yu4xvLnVLkEvCqPVyr0IQMxwJsx4bGLO3cRnUjgT_8w-DfdhO=s0)
+
+5. Once you have created your account, it will be necessary to refresh your page on Dashboard.ata.network. This will then prompt you to authorise polkadot{.js}, which you should approve. *What the Authorize prompt looks like.*
+
+   ![img](https://lh4.googleusercontent.com/5CMLfvyJUYByNx2fByVa3r-BY50LEoKJQmbjSEK1NYTzT8AVdXFQz0uQKnNw3IqaqJOSKfGDosye0V41uIeN9Lrl48r_5r72JJqE7JTRLcjmsTB9UdocQaJYoapgmVmMHd3wWKLd=s0)
+
+6. After you have allowed the application access, you should then click on the settings tab at the top of the screen. 
+
+   *Click on Settings.*
+   
+   ![img](https://lh6.googleusercontent.com/WYw9fBVCaqQ_FyZ4Oa6etI24YFFcNt-trUJQj5y7cPYtnUh8w48D1a5tkf00mkPC2JQusdSX3gy_OaQRFceT_4Q3pFCHBlsOZYXuuHSJX0OyxdZBcABexhahdzi2IIFvYm--HwwN=s0)
+   
+   Then click on Metadata.
+   
+   ![img](https://lh5.googleusercontent.com/d-DVJ8wnEn55ak2BKS6mwv7fDgjLVIbJZFVJNhQkhVlP8WTkqD6oNp6JbFoGKhXYbQsQ4rPpgwn3yNOiqRtybdznFuy6kZH9TWqJT_4xSHr98U20xBOq5ALbNCLjg1C1PdnzGMxI=s0)
+   
+   Then click on upload metadata.
+   
+   ![img](https://lh3.googleusercontent.com/I6p6zxZh2_-pyr9fD6wEjKy_TlO_AnBjC8cROMWn6TF0RrUAjaA7WZ8BVV_AzzN442YuKPrj2RD-6Tl8Zm7HMUBg0eAp9vtRcOW7ViD_pPgxD9G2IJYugpmh09OBmCz5xAKH9Fvj=s0)
+   
+   This will give you another prompt from polkadot{.js} asking to add the metadata to your extension instance. Click ‘yes, do this metadata update’
+   
+   ![img](https://lh6.googleusercontent.com/0ypfBXLgfXbMQFoMpFM0VoJprtMfm38gW_Q3mir_qJdxs7v5n36bb4q7iXEbRv56a2doVk8qMG8c_rcnVTl05KysaeOWbjWdTtX23tzhEfNSqUOXIJmj4PjxbG5plCFtCDv-GC-O=s0)
+
+7. After you have done this, click on the polkadot{.js} plugin again. After this click on the three dots next to your account, click on the drop down box which currently says ‘Allow Use On Any Chain’ and change it to ‘ContextFree Network.’
+
+   ![img](https://lh3.googleusercontent.com/wt48PW1NI3fg-ITzV5YDt8KfpVozZhr_cCbHLS35aIS6PJR99NN8jr4HaJWp3fMPzMLcNvtrmCjmzNkr9d6hdMM3x5neXJScA7-9Z0xfzhJ7ZOBH8ceAkEYmU6MMfss8LiAI1jvq=s0)
+
+   After you have done this to make sure everything is completed click on Accounts again and if there is no exclamation mark next to your account then everything    should be set up.
```

Comment:
This may need to refresh the page

Reply:
Rephrased pt 7. to include "refresh the page" in the guide.